### PR TITLE
Remove (unused) listener module to increase code coverage

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
# Description

Remove (unused) listener module to increase code coverage

### Security checks
 - [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by Chuhaa — APPROVED** [review]

**Comment by Chuhaa** [comment]
Shall we remove listener samples as well?

**Comment by SkNuwanTissera** [comment]
> Shall we remove listener samples as well?

Removed

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -11,15 +11,6 @@ jobs:
               uses: actions/setup-java@v1
               with:
                   java-version: 11
```
Comment (LakshanSS) [inline .github/workflows/pull-request.yml:13]: This is also not needed

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -15,15 +15,6 @@ jobs:
               uses: actions/setup-java@v1
               with:
                   java-version: 11
```
Comment (LakshanSS) [inline .github/workflows/release.yml:17]: Can remove this too

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (SkNuwanTissera) [reply .github/workflows/pull-request.yml]: Updated the PR

File: .github/workflows/release.yml (hunk 2 above)
Comment (SkNuwanTissera) [reply .github/workflows/release.yml]: Updated the PR

**Comment by LakshanSS** [comment]
https://github.com/ballerina-platform/module-ballerinax-googleapis.drive/blob/accdcbcefde17f799379f711e3d18aecf2a6e6ff/settings.gradle#L2

Shall we remove java-wrapper from here too?

**Review by LakshanSS — APPROVED** [review]
Approved with minor suggestions

**Pull request merged**